cli/pack/systemd_dir.go

```diff
@@ -217,7 +226,90 @@ func getSystemdTemplate(ctx *context.Ctx) (templates.Template, error) {
 	return &systemdFilesTemplate, nil
 }
 
-func getSystemdCtx(ctx *context.Ctx, systemdUnitParams SystemdUnitParams) *map[string]interface{} {
+func setDefaultTarantoolEnvValues(ctx *context.Ctx, systemdCtx *map[string]interface{}) {
+	(*systemdCtx)["UnitEnv"] = map[string]interface{}{
+		"TARANTOOL_APP_NAME":     ctx.Project.Name,
+		"TARANTOOL_WORKDIR":      project.GetInstanceWorkDir(ctx, "default"),
+		"TARANTOOL_CFG":          ctx.Running.ConfPath,
+		"TARANTOOL_PID_FILE":     project.GetInstancePidFile(ctx, "default"),
+		"TARANTOOL_CONSOLE_SOCK": project.GetInstanceConsoleSock(ctx, "default"),
+		"TARANTOOL_NET_MSG_MAX":  defaultNetMsgMax,
+	}
+
+	(*systemdCtx)["InstUnitEnv"] = map[string]interface{}{
+		"TARANTOOL_APP_NAME":     ctx.Project.Name,
+		"TARANTOOL_WORKDIR":      project.GetInstancePidFile(ctx, instanceNameSpecifier),
+		"TARANTOOL_CFG":          ctx.Running.ConfPath,
+		"TARANTOOL_PID_FILE":     project.GetInstancePidFile(ctx, instanceNameSpecifier),
+		"TARANTOOL_CONSOLE_SOCK": project.GetInstanceConsoleSock(ctx, instanceNameSpecifier),
+		"TARANTOOL_NET_MSG_MAX":  defaultNetMsgMax,
+	}
+
+	(*systemdCtx)["StateboardUnitEnv"] = map[string]interface{}{
+		"TARANTOOL_APP_NAME":     ctx.Project.StateboardName,
+		"TARANTOOL_WORKDIR":      project.GetStateboardWorkDir(ctx),
+		"TARANTOOL_CFG":          ctx.Running.ConfPath,
+		"TARANTOOL_PID_FILE":     project.GetStateboardPidFile(ctx),
+		"TARANTOOL_CONSOLE_SOCK": project.GetStateboardConsoleSock(ctx),
+		"TARANTOOL_NET_MSG_MAX":  defaultNetMsgMax,
+	}
+}
+
+func setUnitEnv(unitEnv *map[string]interface{}, envParams map[string]interface{}) error {
```

Comment:
What about making the function name more meaningful? Like `updateUnitEnvBySpecifiedArgs`?

Reply:
BTW, maybe declare a `type UnitEnvArgs map[string]interface{}`?